package.json

```diff
@@ -245,5 +246,6 @@
         "expo-font"
       ]
     }
-  }
+  },
+  "packageManager": "yarn@4.10.3+sha512.c38cafb5c7bb273f3926d04e55e1d8c9dfa7d9c3ea1f36a4868fa028b9e5f72298f0b7f401ad5eb921749eb012eb1c3bb74bf7503df3ee43fd600d14a018266f"
```

Comment:
Is this change intended?

Reply:
Ah, not required. Not sure how it got added. Maybe I did `yarn install`.